skydance/protocol.py

```diff
@@ -378,6 +379,31 @@ def __init__(self, raw: bytes):
         """
         self.raw = raw
 
+        lbody = self.body
+        li = 0
+
+        self.deviceType = lbody[li: li + 3]
+        li += 3
+
+        self.scrAddr = struct.unpack("<H", lbody[li: li + 2])[0]
```

Comment:
Is it possible to follow Python naming convention here please? `srcAddr` -> `src_addr`

Reply:
Yes i will fix it, to clarify to me, you doesnt accept camel case, just snake case?(https://khalilstemmler.com/blogs/camel-case-snake-case-pascal-case/)